app/src/main/java/org/eyeseetea/malariacare/database/iomodules/dhis/importer/PullController.java

```diff
@@ -170,12 +169,7 @@ public void pull(Context ctx) {
             //Pull new metadata
             postProgress(context.getString(R.string.progress_pull_downloading));
             try {
-                if(AppSettingsBuilder.isDownloadOnlyLastEvents()){
```

Comment:
the isDownloadOnlyLastEvents functionality should not be deleted, we should download the last event filtering by dates